pycalphad/model.py

```diff
@@ -728,20 +728,30 @@ def magnetic_energy(self, dbe):
         self.TC = self.curie_temperature = tc
         #print(tc)
         # 0.01 used to prevent singularity
```

Comment:
Out of date comment (I believe this was from when this was calculated as Piecewise in temperature)